docs/docs/30-administration/22-backends/40-kubernetes.md

```diff
@@ -1,51 +1,72 @@
 # Kubernetes backend
 
-:::caution
-Kubernetes support is still experimental and not all pipeline features are fully supported yet.
+The kubernetes backend executes steps inside standalone pods. A temporary PVC is created for the lifetime of the pipeline to transfer files between steps.
```

Comment:
I would still call it experimentall or clearly point out, that services dont worl properly jet!

Reply:
And link to https://github.com/woodpecker-ci/woodpecker/issues/1513

Reply:
Hmm in my perception "experimental" is a different state than what the k8s backend is in. 

I am aware of #1513 and happy to link it but I think it's out of the "experimental" state and in a state in which some features are still missing or could be improved but I would label it at least "beta"?

Reply:
Well lets call it beta untill we got it 100% feature compatible with the docker one

Reply:
Well I personnaly would be fine with just clearly mention its current missing feature clearly

Reply:
Sufficient now?

Reply:
Thanks :)